src/client/actions/search.js

```diff
@@ -18,8 +18,8 @@ export const performSearch = searchText => ({
   searchText
 });
 
-export const setSearchError = errorMessage => ({
-  type: SET_ERROR,
+export const receiveSearchError = errorMessage => ({
+  type: RECEIVE_SEARCH_ERROR,
```

Comment:
make this consistent with CLEAR_ERROR, either make both types CLEAR_SEARCH_ERROR/RECEIVE_SEARCH_ERROR or CLEAR_ERROR/RECEIVE_ERROR.

I suggest RECEIVE_ERROR and CLEAR_ERROR